### ISSUE TITLE
cog fails to install using cli

### ISSUE THREAD

**Issue by brian316** [replicate/cog#1914]
When this prompt shows up i click enter (assuming that `/usr/local/bin` is the default)

```shell
sh <(curl -fsSL https://cog.run/install.sh)
Install location? [/usr/local/bin]:
```

what happens:
```shell
The directory  does not exist. Please create it and re-run this script.
```

what i expect is to use the default shown path by clicking enter but i had to manually type it (lazy i know)
can we make it enter for default?

**Comment by Sealical** [comment]
you can take `/usr/local/bin` as input

**Comment by brian316** [comment]
> you can take `/usr/local/bin` as input

yes I know. I mentioned I can type it in. my point was that it is a small detail but was unintuitive
